tools/migration/db/util/mysql_pgsql_1_5_0.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+source $PWD/db/util/pgsql.sh
+source $PWD/db/util/alembic.sh
+
+DBCNF="-hlocalhost -u${DB_USR}"
+
+function mysql_2_pgsql_1_5_0 {
+    
+    alembic_up mysql '1.5.0'
+
+    ## dump 1.5.0-mysql
+    mysqldump --compatible=postgresql --no-create-info --complete-insert --default-character-set=utf8 --databases registry > /harbor-migration/db/schema/registry.mysql
+
+    ## migrate 1.5.0-mysql to 1.5.0-pqsql.
+    python /harbor-migration/db/util/mysql_pgsql_data_converter.py /harbor-migration/db/schema/registry.mysql /harbor-migration/db/schema/registry_insert_data.pgsql
+
+    ## import 1.5.0-pgsql into pgsql.
+    psql -U $1 -f /harbor-migration/db/schema/registry_create_tables.pgsql
+    psql -U $1 -f /harbor-migration/db/schema/registry_insert_data.pgsql
+
+}
+
+# This function is only for <= 1.5.0 to migrate notary db from mysql to pgsql.
+function up_notary {
+
+    set +e
+    if [[ $(mysql $DBCNF -N -s -e "select count(*) from information_schema.tables \
+                where table_schema='notaryserver' and table_name='tuf_files'") -eq 0 ]]; then
+        echo "no content trust data needs to be updated."
+        return 0
+    else
+
+        ## it's not a clean notary db, so cannot execute the create tables step.
```

Comment:
Doesn't Harbor registry DB has the same issue?

Reply:
no, the init() handles the repeats run.